### ISSUE TITLE
Write an update form [UI only]

### ISSUE THREAD

**Issue by blushi** [regen-network/regen-web#2226]
**Describe the solution you'd like**
As a project admin, I can write or edit updates with:
- title of update
- update in markdown (only bold/italic/links supported)
- photos (optional)

https://www.figma.com/file/Bksz1JeDYxQVIXdI46EgPT/Project-Updates?type=design&node-id=19-21002&mode=design&t=ua3Z0u8Dvs7PYEfH-0
____

#### For Admin Use

- [x] Not duplicate issue
- [x] Appropriate labels and zenhub epics applied
- [ ] Appropriate contributors tagged

**Comment by blushi** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/RND-Dev-Team-5f8998bec8958d000f4609e2/issues/regen-network/regen-web/2226?planning-poker) with Zenhub @blushi @flagrede

**Comment by erikalogie** [comment]
@blushi I've updated the Figma links

**Comment by blushi** [comment]
Thanks @erikalogie, since it's just implementing the UI for this form, I was thinking that the "attach a signature" and "congrats" popup could be implemented as part of a follow-up issue that would be about wiring this up in the app once the back-end is ready.

**Comment by blushi** [comment]
Unblocking this, design's ready cc/ @flagrede

**Comment by erikalogie** [comment]
> Thanks @erikalogie, since it's just implementing the UI for this form, I was thinking that the "attach a signature" and "congrats" popup could be implemented as part of a follow-up issue that would be about wiring this up in the app once the back-end is ready cc/ @flagrede

Ah ok, sounds good. FYI @flagrede we discussed on our design huddle yesterday not to implement the controls for bold, italics, and links in this first version, and rather just render any links (http://...) as links for now to save time. I made a dev note about this in the designs, but there should be a follow up task to add the links. cc @aaronc
